databuilder/databuilder/extractor/dbt_extractor.py

```diff
@@ -196,43 +200,87 @@ def _default_sanitize(self, s: str) -> str:
             s = s.lower()
         return s
 
-    def _remove_empty_keys(self, d: Dict) -> Dict:
+    def _get_table_descriptions(self, manifest_content: Dict) -> Tuple[Optional[str], Optional[str]]:
         """
-        Removes all keys in a dictionary where the value is empty or null
+        Gets a description and description source for a table.
         """
-        return {k: v for k, v in d.items() if v}
+        desc, desc_src = None, None
+        if self._extract_descriptions:
+            desc = manifest_content.get('description')
+            desc_src = 'dbt description'
+        return desc, desc_src
+
+    def _get_table_tags_badges(self, manifest_content: Dict) -> Tuple[Optional[List[str]], Optional[List[str]]]:
+        """
+        Gets tags or badges for a given table. At most one of these values will not be null.
+        """
+        tags, tbl_badges = None, None
+        if self._extract_tags:
+            if self._dbt_tag_as == DBT_TAG_AS.BADGE:
+                tbl_badges = manifest_content.get('tags')
```

Comment:
why tbl_badges the same as tags ?

Reply:
This was a request from some members of the community that I was collaborating with. The original implementation was to only use badges since the dbt tags (IMHO) behave more closely to Amundsen badges rather than Amundsen tags but after the feedback we made it configurable with the default being dbt tags are converted to Amundsen badges.